nginx-conf/nginx.conf

```diff
@@ -1,154 +1,196 @@
-            deny all;
-            return 404;
+        if ($request_uri ~ /cookies) {
+            set $access_denied 0;
         }
-
-        # Determine HTTP host to use based on forwarding headers
-        set $customhost $http_host;
-        if ($http_x_forwarded_host != '') {
-            set $customhost $http_x_forwarded_host;
+        if ($request_uri ~ /user) {
+            set $access_denied 0;
         }
-        if ($http_x_original_host != '') {
-            set $customhost $http_x_original_host;
+        if ($request_uri ~ /visit-us) {
+            set $access_denied 0;
         }
-
-        # In Drupal 8, we must also match new paths where the '.php' appears in
-        # the middle, such as update.php/selection. The rule we use is strict,
-        # and only allows this pattern with the update.php front controller.
-        # This allows legacy path aliases in the form of
-        # blog/index.php/legacy-path to continue to route to Drupal nodes. If
-        # you do not have any paths like that, then you might prefer to use a
-        # laxer rule, such as:
-        #   location ~ \.php(/|$) {
-        # The laxer rule will continue to work if Drupal uses this new URL
-        # pattern with front controllers other than update.php in a future
-        # release.
-        location ~ \.php(/|$) {
-            # If we're outside of govuk then request no indexing, on the assumption that this is a non-production environment
-            if ($customhost !~* ".*\.gov\.uk") {
-                add_header X-Robots-Tag "noindex, follow" always;
-            }
-
-            # Override host for health endpoint
-            if ($uri = "/index.php/health")  {
-                set $customhost drupal;
-            }
-            fastcgi_split_path_info ^(.+?\.php)(|/.*)$;
-            include fastcgi_params;
-            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-            fastcgi_param PATH_INFO $fastcgi_path_info;
-            fastcgi_param HTTP_HOST $customhost;
-            # make drupal understand clients were talking https to the loadbalancer
-            fastcgi_param HTTPS on;
-            fastcgi_param HTTP_SCHEME https;
-            fastcgi_intercept_errors on;
-            # changed from drupal-fpm to localhost for azure containers
-            fastcgi_pass localhost:9000;
-            fastcgi_buffers 16 32k;
-            fastcgi_buffer_size 64k;
-            fastcgi_busy_buffers_size 64k;
+        if ($request_uri ~ /csearch) {
+            set $access_denied 0;
         }
-
-        location ~* \.(js|css|png|jpg|jpeg|gif|ico|svg)$ {
-            try_files $uri @rewrite;
-            expires max;
-            log_not_found off;
+        if ($request_uri ~ /08b5d5dc-f2de-4c78-86a7-d3ea80037430) {
```

Comment:
Just curious what this is for?

Reply:
That's the health-check url that gets pinged externally